### PR TITLE
Grammar fixes and rewording

### PR DESCRIPTION
**Pull request by airandfingers**
Fixes for the "Data and bindings in views" learn page

### CODE REVIEW

File: learn_markdown/data-bindings-in-views.md [hunk 1]
```diff
@@ -5,19 +5,19 @@ order: 4
...
+We may want to render the page and get pixels on the screen as soon as possible, but what happens when we don't have all our data yet? Somehow we have to coordinate this, so that the view is updated as relevant data arrives or properties change.
 
 Ampersand-view handles this with declarative bindings. The binding declarations are *completely* decoupled from the data models.
 
-In the same way you declare a hash of events to listen to from the DOM, ampersand-view lets you declare a hash of bindings to go the other way: from your models to the DOM. You explicitly describe the relationship and then the view handles the rest, no matter when you get your models or data.
+ampersand-view, like Backbone.View, accepts a hash of events to listen to from the DOM, but ampersand-view also lets you declare a hash of bindings to go the other way: from your models to the DOM. You explicitly describe the relationship and then the view handles the rest, no matter when you get your models or data.
```
Comment (wraithgar) [inline learn_markdown/data-bindings-in-views.md]: I don't think we need to throw back to backbone here, ampersand-view has an events hash and that is what's being referenced here.

File: learn_markdown/data-bindings-in-views.md [hunk 2]
```diff
@@ -5,19 +5,19 @@ order: 4
...
 
-This gives you complete flexibility over when you render what.
+This gives you complete flexibility over what, how, and when you render.
 
-You can most certainly use whatever template language you want with ampersand-views, but the nice things is **no template engine is actually necessary in order to get bindings**!
+You can use whatever template engine you want with ampersand-view bindings, or **no template engine at all**!
 
-You can simply have an HTML string as your `template` property and declare your bindings and everything Just Works™.
+You can simply have an HTML string as your `template` property and declare your bindings and Everything Just Works™.
```
Comment (wraithgar) [inline learn_markdown/data-bindings-in-views.md]: this is an allusion to it Just Works and the lowercase e was on purpose.

**Comment by wraithgar** [comment]
Only a couple of nitpicks, otherwise all good changes.

File: learn_markdown/data-bindings-in-views.md (hunk 2 above)
Comment (airandfingers) [inline learn_markdown/data-bindings-in-views.md]: Ah, I wasn't familiar with that slogan! Lowercase e it is.

File: learn_markdown/data-bindings-in-views.md (hunk 1 above)
Comment (airandfingers) [inline learn_markdown/data-bindings-in-views.md]: Okay, the hash wasn't mentioned previously in the documentation, and I've used Backbone, so "like Backbone _plus_" was the impression I got from this sentence.

**Comment by wraithgar** [comment]
This all reads much nicer now, thank you.

**Pull request merged**